driver-core/src/main/com/mongodb/internal/connection/LoggingCommandEventSender.java

```diff
@@ -133,53 +158,82 @@ public void sendFailedEvent(final Throwable t) {
 
     @Override
     public void sendSucceededEvent(final ResponseBuffers responseBuffers) {
-        long elapsedTimeNanos = System.nanoTime() - startTimeNanos;
-
-        if (loggingRequired()) {
-            logger.debug(
-                    format("Execution of command with request id %d completed successfully in %s ms on connection [%s] to server %s",
-                            message.getId(), getElapsedTimeFormattedInMilliseconds(elapsedTimeNanos), description.getConnectionId(),
-                            description.getServerAddress()));
-        }
-
-        if (eventRequired()) {
-            BsonDocument responseDocumentForEvent = redactionRequired
-                    ? new BsonDocument()
-                    : responseBuffers.getResponseDocument(message.getId(), new RawBsonDocumentCodec());
-            sendCommandSucceededEvent(message, commandName, responseDocumentForEvent, description,
-                    elapsedTimeNanos, commandListener, requestContext);
-        }
+        sendSucceededEvent(responseBuffers.getResponseDocument(message.getId(), new RawBsonDocumentCodec()));
     }
 
     @Override
     public void sendSucceededEventForOneWayCommand() {
+        sendSucceededEvent(new BsonDocument("ok", new BsonInt32(1)));
+    }
+
+    private void sendSucceededEvent(final BsonDocument reply) {
```

Comment:
DRY-ed it up given that the spec doesn't differentiate the log messages for one-way command (which are used for unacknowledged writes)